framework/include/utils/MooseUtils.h

```diff
@@ -383,6 +385,23 @@ tokenizeAndConvert(const std::string & str,
   }
   return true;
 }
+
+/**
+ * Convert supplied string to upper case.
+ */
+std::string toUpper(std::string name);
```

Comment:
OK - just so used to this being an oversight. I realize the comment says it converts the input string to upper-case but since it returns it, I thought I'd comment. It might be worth an explicit comment since it does both!